README.md

```diff
@@ -1,44 +1,39 @@
 # Radius Community Recipes
 
-This repo contains commonly used [Recipe](https://docs.radapp.dev/recipes) templates for Radius Environments.
+This repository contains commonly used [Recipe](https://docs.radapp.io/recipes) templates for Radius Environments.
 
 ## Recipes
 
 Recipes provide self-service infrastructure provisioning for developers. Developers select the resource(s) they need, and operators can configure Recipes with secure, approved, infrastructure.
 
-## Available Recipes
-
-### local-dev
+## Supported Resources types
 
-The [local-dev](/local-dev) directory contains lightweight Recipes for development purposes. They run containerized infrastructure which is not persisted across restarts and is optimized for CPU and memory usage on a local machine.
+Recipes currently support the following resources:
```

Comment:
Instead of a hard coded list here, I think it would be easier to maintain it by providing a link to the available portable resources page: https://docs.radapp.io/guides/author-apps/portable-resources/overview/#available-resources?

Reply:
Idea is to let the user know the existing support upfront without having to click. How often do you think this list will change?